source/administration/command-line-tools.rst

```diff
@@ -112,6 +113,23 @@ platform channel add
 
       sudo ./platform channel add mychannel user@example.com username
 
+platform channel archive
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Archive a channel. Archived channels are not accessible to users, but remain in the database. To restore a channel from the archive, see `platform channel restore`_.
+
+  Format
+    .. code-block:: none
+
+      platform channel archive
```

Comment:
Should there be "{channel}" or  "{team}:{channel}" in the format block? See format and description for restoring and it should be consistent